cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,8 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
-REGISTRY?=gcr.io
-PROJECT?=google_containers
+REGISTRY ?= gcr.io
```

Comment:
Did this get lost in a rebase? Should this be staging-k8s.gcr.io?

Reply:
ACK, pushed!  Good catch.